binaries/workerserver/main.go

```diff
@@ -16,13 +15,13 @@ import (
 	"github.com/scootdev/scoot/workerapi/server"
 )
 
-var thriftPort = flag.Int("thrift_port", 9090, "port to serve thrift on")
-var httpPort = flag.Int("http_port", 9091, "port to serve http on")
+var thriftAddr = flag.String("thrift_addr", "localhost:9090", "port to serve thrift on")
```

Comment:
Little out of scope but we may want to start defining and using ports via single point of truth package var